### ISSUE TITLE
The FONT (face, size or color) html tags appear to not work.

### ISSUE THREAD

**Issue by lwhistler** [reingart/pyfpdf#37]
All the HTML tags work except for the FONT(face, size or color). Is there sample syntax I can view that changes the font size and face? Particularly inside table cells. Thanks

**Comment by vyscond** [comment]
for now just give up on this. the html render function is too much complex man.

**Comment by lwhistler** [comment]
OK .... Thanks for the quick reply.  My project is manipulating a CSV file and outputting directly to a PDF file. I would like to skip the spreadsheet step. It's 100% working with alternating row colors, page numbering, header, and 10 columns. It can process a 5000 line CSV file in 10 seconds to a 157 page PDF file, ready for printing. 

The problem is the default font in the HTML table is a tad too big for my 10 column layout in Portrait format. So I have to use Landscape format instead. If I could scale down the font size then it will fit the Portrait format, which I prefer.

Thanks again for the reply - and I really like the idea of using a HTML layout to output a PDF file using Python.

**Comment by vyscond** [comment]
Yeah. I understand too, but i think know that convert html to PDF is quite the same problem of read html and render on a browser. So it's not really simple to read html tags with a low time response.

what I suggest for you is to spent some time building your own table write function using the __write__ functions. This is what i choose to do because didn't want to write an specific backend on another language with more pdf generation support, or even handle system binaries to render it.

PyPDF is a pure python option which provides you the possibility to not ask for help from any other tech. Off course it's not have all the magic functions we want, but it's a about time.

If you don't have time to think too much about how you can draw a flawless table with the available functions you can think about using the (payed reportlab version)[http://www.reportlab.com/] which provides a DSL to work with PDF.

But if you decide to building the ultimate table draw function just give a signal, the community will help! :)

**Comment by lwhistler** [comment]
I got the problem solved - I hard coded the font size and color in the html.py file.

**Comment by vyscond** [comment]
... or this.
